### DISCUSSION TITLE
Axis OS 12 and root dependencies

### DISCUSSION THREAD

**Discussion by sebastianengman** [Mo3he/Axis_Cam_Tailscale#11] [General]
Hi. 

@Mo3he what is your take on the possibilities to add cloud connectivity in the future, when root access is not available in ACAP?

My knowledge is limited on the subject, but I am however interested in a solution for bringing cloud connectivity to, for example, Axis cameras. 

Would for example Cloudflare (Tunnel) require root access to work?

**Comment by Mo3he** [comment]
We have found a way to run without root and will support OS 12! More updates soon.
